tests/PHPStan/Analyser/LegacyNodeScopeResolverTest.php

```diff
@@ -1129,19 +1129,19 @@ public function dataArrayDestructuring(): array
 				'$fourthStringArrayForeachList',
 			],
 			[
-				'string',
+				'lowercase-string',
```

Comment:
I don't get this change. IMHO it should remain a string.

Reply:
Or maybe it should be numeric-string.

Reply:
`'2018-12-19'` is technically a lowercase-string.

And `explode($lowercaseString)` is `array<lowercase-string>`.

So every element of the array is precised from `string` to `lowercase-string`.